src/screens/CatalogScreen.js

```diff
@@ -1,28 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
   StyleSheet, SafeAreaView, StatusBar, ScrollView,
 } from 'react-native';
+import { connect } from 'react-redux';
 
+import { shape } from 'prop-types';
 import TimelineWidget from '../components/Timeline/TimelineWidget';
 import CategorySelector from '../components/CategorySelector/CategorySelector';
-import RecordCardsContainer from '../components/RecordCard/RecordCardsContainer';
+import RecordCardsContainer from '../components/SubTypeAccordion/SubTypeAccordionsContainer';
 import Colors from '../constants/Colors';
 import FilterDrawer from '../components/FilterDrawer/FilterDrawer';
 
-const CatalogScreen = () => (
-  <SafeAreaView style={styles.safeAreaView}>
-    <StatusBar backgroundColor={Colors.primary} barStyle="dark-content" />
-    <FilterDrawer>
-      <ScrollView style={styles.content}>
-        <TimelineWidget />
-        <CategorySelector />
-        <RecordCardsContainer />
-      </ScrollView>
-    </FilterDrawer>
-  </SafeAreaView>
-);
+const CatalogScreen = ({ resourceIdsGroupedByType }) => {
+  const categories = Object.keys(resourceIdsGroupedByType);
+  const scrubbedCategories = categories.filter((category) => category !== 'Patient' && category !== 'Observation');
```

Comment:
`resourceIdsGroupedByType`, from the selector, is already filtered by valid categories (and omitting Patient)

Reply:
this is addressed in #29